frontend/src/core/loaders/components/SkeletonLoader.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import './SkeletonLoader.css';
+
+const list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+const SkeletonLoader = () => (
+    <div className="skeleton-loader entities unselectable">
```

Comment:
You can remove the `entities unselectable` classes.

Reply:
Hi, @mathjazz I noticed that when I removed the entities class, the list does not load, it just stops on the skeleton page. 
I can't really figure out why it's behaving like that.